Platformgame/Assets/0. KimJiMin/MissileCtrl.cs

```diff
@@ -1,11 +1,15 @@
-﻿using System.Collections;
+
+﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 // written by Lim Yeonsang
 public class MissileCtrl : MonoBehaviour
 {
-    private float _bombSpeed = 1.5f; //미사일 속도
+    public float _bombSpeed; //미사일 속도
+    public float MissileReferenceTime; //미사일 생성 주기
+    public GameObject Missile; //미사일 프리팹
     private readonly Vector3 _direction = Vector3.down;
 
```

Comment:
```suggestion
    private float _bombSpeed = 1.5f; //미사일 속도
    public float _bombSpeed; //미사일 속도
    public float MissileReferenceTime; //미사일 생성 주기
    public GameObject Missile; //미사일 프리팹
    private readonly Vector3 _direction = Vector3.down;
```
`private, public 변수 선언은 나누어 정리` 규칙 및 `변수 이름은 소문자로, 메서드와 클래스 이름은 대문자로 시작한다.` 규칙이 지켜지지 않았습니다.